db/migration/migrations/000015_add_request_version_availability.up.sql

```diff
@@ -0,0 +1,41 @@
+BEGIN;
```

Comment:
The .up migration should include adding the correct requested versions for the fhir_endpoints_metadata table using the info history table to get the requested version corresponding to those entries, and for the fhir_endpoints_availability table, should add "None" as the default requested version.

Reply:
Wow.... how did I miss this? Very good catch.